Library/Formula/flatcc.rb

```diff
@@ -0,0 +1,54 @@
+class Flatcc < Formula
+  desc "FlatBuffers Compiler and Library in C for C"
+  homepage "https://github.com/dvidelabs/flatcc"
+  url "https://github.com/dvidelabs/flatcc/archive/v0.2.0.tar.gz"
+  sha256 "5327e0c0faa37643f59be6f086576ef76a13d36b655d2bf9547f89ffce3b3413"
+  head "https://github.com/dvidelabs/flatcc.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-G", "Unix Makefiles", buildpath, *std_cmake_args
+    system "make"
+
+    bin.install "bin/flatcc"
+    lib.install "lib/libflatcc.a"
+    lib.install "lib/libflatccrt.a"
+    include.install Dir["include/*"]
+  end
+
+  test do
+    def testfbs; <<-EOS.undent
```

Comment:
This can just be `(testpath/"test.fbs").write <<-EOS.undent` instead of having a one-off method.